.github/workflows/autobuild.yaml

```diff
@@ -160,15 +160,16 @@ jobs:
           os: macos-latest
           preinstall: brew install ninja zlib p7zip
           cflags: -O3 -gline-tables-only -DNDEBUG
-          env: MACOSX_DEPLOYMENT_TARGET=10.9
           cmake: >-
             "-DCMAKE_C_COMPILER=clang"
             "-DCMAKE_CXX_COMPILER=clang++"
             "-DLLVM_ENABLE_ZLIB=FORCE_ON"
             "-DCMAKE_OSX_ARCHITECTURES=x86_64;arm64"
+            "-DCMAKE_OSX_DEPLOYMENT_TARGET=10.9"
           # BoringSSL doesn't support universal binaries when building with ASM.
           grpc_cmake: >-
             "-DOPENSSL_NO_ASM=ON"
+            "-DCMAKE_OSX_DEPLOYMENT_TARGET=10.9"
```

Comment:
I don't think you need it here, $grpc_cmake is *additions* to $cmake.

Reply:
right, thanks!